modules/misc/nix.nix

```diff
@@ -10,6 +10,27 @@ let
 
   isNixAtLeast = versionAtLeast (getVersion nixPackage);
 
+  nixPath = lib.concatStringsSep ":" cfg.nixPath;
+
+  useXdg = config.nix.enable
+    && (config.nix.settings.use-xdg-base-directories or false);
+  defexprDir = if useXdg then
+    "${config.xdg.dataHome}/nix/defexpr"
```

Comment:
Isn't it supposed to be `XDG_STATE_HOME` (according to [the documentation](https://nixos.org/manual/nix/stable/command-ref/files/default-nix-expression))?.

Reply:
Thanks, fixed. I was looking at the outdated PR description in https://github.com/NixOS/nix/pull/5588.